<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_UI</class>
 <widget class="QWidget" name="Edit_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>335</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit_UI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Tag Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Tag_Name"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Attribute Names</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="attributeNames"/>
         </item>
         <item>
          <widget class="QComboBox" name="attributeCombo"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Tag Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Tag_Value"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Attribute Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Att_Value"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
